Dockerfile

```diff
@@ -17,7 +18,26 @@ RUN buildDeps='libmagic-dev' && \
     rm -rf /var/lib/apt/lists/* && \
     bundle install --deployment --without development test && \
     apt-get purge -y --auto-remove $buildDeps && \
-    whenever --update-crontab
+    echo "Foodsoft::Application.config.secret_token = ENV['SECRET_KEY_BASE']" > config/initializers/secret_token.rb && \
```

Comment:
Can we do this in the source repository and use `.env`? It's good practice anyway, and makes deployment with other methods easier too.